### DISCUSSION TITLE
Model의 `__str__`을 사용하는 이유? (용도)

### DISCUSSION THREAD

**Discussion by devnproyj22** [weaverse-techtide/weaverse-backend#108] [General]
예. 테스트에서 breakpoint로 사용하기 위함
예. 파이썬의 기본

**Comment by Yoo117** [comment]
결국은 관리자 측면에서 도움을 주기 위한 것이라고 생각합니다. 관리자 페이지(혹은 대시보드)에서 객체를 쉽게 이해하고 관리할 수 있게 해주며, 로그나 에러 메세지에서 직관적으로 뜨는 걸 통해 디버깅에 도움을 준다고 봅니다.
